src/less/variables/ds6/icon-variables.less

```diff
@@ -167,12 +171,18 @@
 @icon-profile-height: 24px;
 @icon-clock-width: 22px;
 @icon-clock-height: 22px;
-@icon-radio-checked-width: 24px;
-@icon-radio-checked-height: 24px;
-@icon-radio-checked-small-width: 16px;
-@icon-radio-checked-small-height: 16px;
-@icon-radio-unchecked-width: 24px;
-@icon-radio-unchecked-height: 24px;
+@icon-radio-checked-width: 18px;
```

Comment:
What is the point in having both small and default sizes if they're the same? 
If the regular ones will be removed at a later date, maybe just alias them and put a comment to do so.

Reply:
Yeah, good point, I'll just alias them here.